packages/extension-dapp/src/index.ts

```diff
@@ -46,13 +52,14 @@ export { isWeb3Injected, web3EnablePromise };
 
 function getWindowExtensions (originName: string): Promise<[InjectedExtensionInfo, Injected | void][]> {
   return Promise.all(
-    Object.entries(win.injectedWeb3).map(([name, { enable, version }]): Promise<[InjectedExtensionInfo, Injected | void]> =>
-      Promise.all([
-        Promise.resolve({ name, version }),
-        enable(originName).catch((error: Error): void => {
-          console.error(`Error initializing ${name}: ${error.message}`);
-        })
-      ])
+    Object.entries(win.injectedWeb3).map(
```

Comment:
No logic changes, only reformatting. In this case the polkadot-js style is actually very different. In these types of situations, the chain is actually split if needed. Would just suggest to revert.

Reply:
Still applicable, but will do a manual revert of this and the one above once in.